File: Lab nertwork java/Lab1/ServerServices/src/main/java/services/IServices.java
package services;
import models.*;

public interface IServices {
    void login(DTOAngajat angajat, IObserver client) throws ServerException;
    void logout(DTOAngajat angajat, IObserver client) throws ServerException;
    void submitInscriere(DTOInfoSubmit infoSubmit) throws ServerException;
    //Oficiu[] getLoggedEmployees() throws ServerException;
    CursaDTO[] getCurseDisp() throws ServerException;
    ParticipantCursaDTO[] searchByTeam(String team) throws ServerException;

}

File: Server/Services/IServices.cs

using Model.Model;
using Problem11.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Services
{
    public interface IServices
    {
        void login(DTOAngajat angajat, Observer client);
        void logout(DTOAngajat angajat, Observer client);
        void submitInsc(DTOInfoSubmit infoSubmit);
        DTOBJCursa[] getCurrentRaces();
        DTOBJPart[] searchbyteam(string team);
    }
}
